pennylane/transforms/decompositions/two_qubit_unitary.py

```diff
@@ -0,0 +1,657 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains transforms and helpers functions for decomposing arbitrary two-qubit
+unitary operations into elementary gates.
+"""
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane import math
+
+from .single_qubit_unitary import zyz_decomposition
+
+
+###################################################################################
+# Developer notes:
```

Comment:
This is the best info I can get right now; it doesn't seem like there is a straightforward fix so I will leave this to future work.

Reply:
This is super useful to have 💯

Reply:
Thank you so much Olivia! This has been very helpful.